internal/deviations/README.md

```diff
@@ -1,21 +1,39 @@
-## Guidelines to add deviations to FNT tests
+# Guidelines to add deviations to FNT tests
 
-### Adding Deviations
+## When to use deviations
+
+1. Deviations may be created to use alternate OC or use CLI instead of OC to achieve the operational intent described in the README.
+2. Deviations should not be created which change the operational intent.  See below for guidance on changing operational intent.
+3. Deviations may be created to change which OC path is used for telemetry or even using a native OC path.  Deviations for telemetry should not introduce CLI depedency.
```

Comment:
By `CLI dependency`, you meant `CLI command output`, right?

Reply:
Yes, thanks.   I've refreshed the text to be more clear.